src/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+# libtool versioning
+# See http://sources.redhat.com/autobook/autobook/autobook_91.html#SEC91 for details
+
+## increment if the interface has additions, changes, removals.
+set(LT_CURRENT 21)
+
+## increment any time the source changes; set 0 to if you increment CURRENT
+set(LT_REVISION 0)
+
+## increment if any interfaces have been added; set to 0
+## if any interfaces have been changed or removed. removal has
+## precedence over adding, so set to 0 if both happened.
+set(LT_AGE 13)
+
+math(EXPR LT_CURRENT_MINUS_AGE "${LT_CURRENT} - ${LT_AGE}")
+
+set(SONAME ${LT_CURRENT_MINUS_AGE}.${LT_AGE}.${LT_REVISION})
+set(SOVERSION ${LT_CURRENT_MINUS_AGE})
+
+if (ENABLE_CCE)
+    add_subdirectory("CCE")
+endif()
+add_subdirectory("common")
+add_subdirectory("CPE")
+add_subdirectory("CVE")
+add_subdirectory("CVSS")
+add_subdirectory("DS")
+add_subdirectory("OVAL")
+add_subdirectory("SCE")
```

Comment:
isn't SCE optional like CCE?

Reply:
@mpreisler Yes, it is. The option is checked in `src/SCE/CMakeLists.txt`. However, I'm going to move the check here, as you suggested below.